options/ansi/include/mlibc/file-io.hpp

```diff
@@ -3,6 +3,7 @@
 
 #include <stdio.h>
 
+#include <mlibc/file.hpp>
```

Comment:
I'm not convinced that this is a good idea. We basically gain nothing here and we lose the ability to use inline functions that access struct file internals (which will be quite a lot faster than PLT calls).